pullrequest.rst

```diff
@@ -105,23 +105,23 @@ Here is a quick overview of you can contribute to CPython on GitHub:
 
 1.  If an Issue doesn't exist, `create an Issue`_ that describes your change
 
-2.  Prepare your system by :ref:`setup`
+2.  :ref:`Get started <setup>`_ and set up your system
```

Comment:
Ah yeah. 😅

The reason I didn't see any errors, is that I failed to check & push in my final change. Thanks again!